spec/models/events/creator_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Events::Creator, :type => :model do
+
+  before do
+    Timecop.freeze(Time.local(1990))
```

Comment:
The tests are important here because the class is implementing logic to set the start date. It's not directly coming from a part of the framework.

https://github.com/alex-hall/connector/blob/93e3c4ea4026ca1f40c945da51f3f5092cc879cb/app/models/events/creator.rb#L25